8.9.1 Release notes (#3690)
elastic/security-docs: docs/release-notes/8.9.asciidoc

[[release-notes-header-8.9.0]]
== 8.9

[discrete]
[[release-notes-8.9.1]]
=== 8.9.1

[discrete]
[[breaking-changes-8.9.1]]
==== Breaking changes

There are no breaking changes in 8.9.1.

[discrete]
[[enhancements-8.9.1]]
==== Enhancements

> * Event correlation queries and rules can now detect missing events in EQL sequences.

Reviewer: @nastasha-solomon if Paul doesn't suggest a PR for us to link to, what do you think about including a cross-ref to our create EQL rule doc link here?

Author: Yeah, good idea. I'll merge this change tomorrow so the link resolves and the CI checks pass:

```suggestion
* Event correlation queries and rules can now detect {ref}/eql-syntax.html#eql-missing-events[missing events] in EQL sequences.
```

Author: As discussed with @chriscressman over slack, I'll remove this ref for now and then re-add it tomorrow in a separate PR.

Author: PR for change: https://github.com/elastic/security-docs/pull/3736

[discrete]
[[bug-fixes-8.9.1]]
==== Bug fixes 

* Fixes a copy to clipboard bug that affected non-ECS fields ({pull}162883[#162883]).

> * Fixes rounding issues in the *Top alerts by* table on the Alerts page ({pull}162647[#162647]).

Reviewer: I would just point out this is related to numbers/decimals because "rounding" can mean several things. 
```suggestion
* Fixes number rounding issues in the *Top alerts by* table on the Alerts page ({pull}162647[#162647]).
```

[[release-notes-header-8.9.0]]
== 8.9

[discrete]
[[release-notes-8.9.1]]
=== 8.9.1

[discrete]
[[breaking-changes-8.9.1]]
==== Breaking changes

There are no breaking changes in 8.9.1.

[discrete]
[[enhancements-8.9.1]]
==== Enhancements

> * Event correlation rules now support negative correlation. 

Author: @paulewing let me know if you'd like for this summary to be expanded and linked to a Security PR - thanks!

> * Event correlation rules can now detect missing events within EQL sequences. 

Author: ```suggestion
* Event correlation queries and rules can now detect missing events in EQL sequences.
```

[discrete]
[[bug-fixes-8.9.1]]
==== Bug fixes 

* Fixes a copy to clipboard bug that affected non-ECS fields ({pull}162883[#162883]).
* Fixes rounding issues in the *Top alerts by* table on the Alerts page ({pull}162647[#162647]).
* Fixes bug that prevented controls from being rendered on {elastic-sec} dashboards ({pull}162514[#162514]).

> * Fixes exceptions in investigate in timeline ({pull}162190[#162190]).

Author: @kqualters-elastic after reading the description, I'm still not quite sure what the bug (or bugs) were. Was this bug related to the "open field in Timeline" action that users could take from anywhere within the Security app?

Reviewer 2: I think this is a little too vague, maybe something like ""Fixes a bug where an alert based on a rule with exceptions could cause timeline to fail to load properly" ?

Reviewer 2: might have to scrap this all together, as the pr has been merged to main but the backport is being blocked by unrelated test failures https://github.com/elastic/kibana/pull/163373

Author: Ok, let me know if I need to remove it from the release notes - thank you!

Reviewer 3: You can remove it @nastasha-solomon, thanks for checking!

Author: ```suggestion
```
